envpool/sokoban/astar_log.cc

```diff
@@ -19,12 +19,13 @@
 
 namespace sokoban {
 
-void RunAStar(std::string file_idx, int fsa_limit = 1000000) {
+void RunAStar(std::string file_idx, int fsa_limit = 1000000,
+              std::string dataset = "train") {
   std::cout << "Running A* on file " << file_idx << " with fsa_limit "
             << fsa_limit << std::endl;
   std::stringstream s1, s2;
-  s1 << "/training/.sokoban_cache/boxoban-levels-master/unfiltered/train/"
-     << file_idx << ".txt";
+  s1 << "/training/.sokoban_cache/boxoban-levels-master/unfiltered/ " << dataset
```

Comment:
Can you make the script just take a path to the `.txt`? We shouldn't have to recompile it if we want to use a path other than `unfiltered`. This stuff really should be in the k8s yaml.

Reply:
That makes sense. Thanks for the suggestion! I've modified it to take in the level and log file names.